docs/source/CppUsage.md

```diff
@@ -88,6 +88,11 @@ convenient accessors for all fields, e.g. `hp()`, `mana()`, etc:
 
 *Note: That we never stored a `mana` value, so it will return the default.*
 
+The following attributes are supported:
```

Comment:
maybe also mention it in the object api section below.

Reply:
Made verbose by adding "object based API" into description to allow text searching.